TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -152,6 +156,21 @@ class BenchmarkParser extends Parser {
                 benchmarkProduct: curBenchmarkProduct,
                 testData: curTestData
             } );
+            // collect info and make calculation based on the curTestData
+            if ( curTestData.metrics && curTestData.metrics.length > 0 ) {
+                for ( let { name, value } of curTestData.metrics ) {
+                    let childValue = value.filter( n => n );
+                    // in child Aggregate data, 'interation' is the number of valid values in each metric that iterates.
+                    childAggData.push({name: name, value: {mean: math.mean(childValue), max: math.max(childValue), min: math.min(childValue), median: math.median(childValue), stddev: math.std(childValue), CI: BenchmarkMath.confidence_interval(childValue), iteration: childValue.length}});
```

Comment:
- please format the code to be more readable. For example, a new line after each `,`
- when key and value are the same, we can write it once. For example, `name: name` can be changed to `name`

Reply:
updated!